src/common/base/abstractPromote.ts

```diff
@@ -107,6 +108,9 @@ export abstract class PromoteCommand<T extends typeof SfCommand> extends SfComma
       this.outputService
     );
     await doceMonitor.monitor();
+    if (this.outputService.getStatus() === AsyncOperationStatus.Completed && this.flags.verbose) {
+      await this.outputService.displayEndResults();
```

Comment:
I think we should let the OuputService decide what to display.  It knows the verbosity level right?  

if (getStatus() === Completed)
  this.outputService.displayEndResults();